Notifier-service/src/models/conference.ts

```diff
@@ -50,11 +50,11 @@ export class ConferenceModelMongo extends ConferenceModel {
     }
 
 
-    async getOne(): Promise<ConferenceDocument | null> {
+    async getOne(query:any = {}): Promise<ConferenceDocument | null> {
```

Comment:
* Fix this as indicated above.

Reply:
Hi, I understand what you're saying. I thought of specifying the type of query as {[key:string]:any} so that getOne method can be used anywhere to query mongo based on any key and value specified rather than just _id. Is that ok?

Reply:
* Don't do that. As I said the model will solely be responsible for exposing what can be done with the data. Any complex query can be embedded inside it. What you are suggesting can be added as a sepparate function like `fromQuery( query: Query )` , . Which can be added later. For now stick to making query as type string. and keeping strict type and functionality focused.

Reply:
Sure, understood, I'll make the change, thanks